### ISSUE TITLE
Resize bug

### ISSUE THREAD

**Issue by boolieman** [futurepress/epub.js#579]
I am writing this inside a React Wrapper and using the refferences to container and iframe in order to update the size when being resized, the problem is that the iframe > document > body has a written style that updates when a user changes location and codes in a hard value for the width of the body, after a single page is navigated it does not allow the width of the body to be updated in a flexible manner.  All it appears i need is to be able to maintain a body key of `width: auto` in order to fix the bug i'm seeing.  Is there a property that i am missing for this? (I have tried a good number of them and even dug around inside the code to find an answer to this)

**Comment by liamgui** [comment]
@boolieman how did you solve this? I am having the same issue right now. I have spent upwards of 10 hours hunting for a solution. I've tried using the rendition.resize but that just breaks everything.

**Comment by kamui-fin** [comment]
Can confirm I'm seeing similar issues too

**Comment by boolieman** [comment]
It's been a while since I found my way around but I'm pretty sure I generally ended up wrapping it inside a react component that has a resize listener and I would denounce that and force a redraw of the whole reader. Like I said. It's been a while. I was doing this for a job back then and we ended up doing a lot of local modifications since we also needed to support IE at the time too which meant we had to do a lot of custom compiling of the code.
